ssm/agents.py

```diff
@@ -104,35 +102,13 @@ def get_ssm_args(protocol, cp, log):
         project = None
         token = ''
 
-        use_ssl = cp.getboolean('broker', 'use_ssl')
-        if use_ssl:
-            service = STOMP_SSL_SERVICE
-        else:
-            service = STOMP_SERVICE
```

Comment:
Is `use_ssl` still used elsewhere? If not, should we remove it from the config files?

Reply:
It can still be used in theory by STOMP messaging.

Reply:
ah this just remove the local `use_ssl` within the `get_ssm_args` method?

Not `use_ssl` more broadly?

Reply:
Yes, `use_ssl` was used for choosing if you fetched SSL or non-SSL brokers from the BDII (using LDAP), but also in the `stomp.Connection` constructor which is still in SSM for now.

Reply:
👍